packages/flutter_tools/gradle/build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
+    id 'groovy-gradle-plugin'
+}
+
+sourceSets {
+    main {
+        groovy {
+            srcDirs += ['.']
+        }
+    }
+}
+
+repositories {
+    mavenCentral()
+    google()
+}
+
+group = 'dev.flutter'
+version = '1.0.0'
+
+gradlePlugin {
+    // Define the plugin
+    plugins {
+        blahblah {
```

Comment:
:slightly_smiling_face: What does Gradle do with this identifier? It looks like you don't end up having to mention it again.

Reply:
Apparently nothing - that's why I named it like this to make it explicit for us. In practice, most plugins do something like `customPlugin` or `${pluginName`}, so we could go with `flutterPlugin`.

Reply:
If the name does not matter please add a comment saying so :)